src/main/kotlin/no/nav/familie/ef/sak/behandlingsflyt/steg/VentePåStatusFraIverksett.kt

```diff
@@ -17,17 +18,16 @@ class VentePåStatusFraIverksett(private val iverksettClient: IverksettClient,
     override fun utførSteg(saksbehandling: Saksbehandling, data: Void?) {
         iverksettClient.hentStatus(saksbehandling.id).let {
             when {
-                erMigreringOgOk(saksbehandling, it) -> opprettLagSaksbehandlingsblankettTask(saksbehandling)
+                skalIkkeSendeBrev(saksbehandling, it) -> opprettLagSaksbehandlingsblankettTask(saksbehandling)
```

Comment:
Usikker på metodenavnet her - sjekken er jo om iverksett er "OK"-ish for ikke-brev-baserte iverksettinger... 
Men litt vrient å finne et bra navn.

`erBrevløsIverksettingOk` ❓

Reply:
Ja, egentlige burde kanskje iverksetting ha denne typen logikken i stedet.. Endret til `erBrevløsIverksettingOk`